File: Learn_Android/getWallpaer/app/src/main/java/com/example/sunkai/getwallpaer/Tools/MakeToast.java
package com.example.sunkai.getwallpaer.Tools;

import android.widget.Toast;

/**
 * Created by sunkai on 2018/1/26.
 */

public class MakeToast {
    public static void Toast(String s){
        Toast.makeText(GlobalContext.context,s,Toast.LENGTH_SHORT).show();
    }
}

File: Learn_Android/MyApplication/app/src/main/java/com/example/sunkai/myapplication/Tools/MakeToast.kt
package com.example.sunkai.myapplication.Tools

import android.app.Activity

/*
 * Created by sunkai on 2018/1/26.
 */

fun Activity.Toast(str:String){
    android.widget.Toast.makeText(GlobalContext.context,str,android.widget.Toast.LENGTH_SHORT).show()
}